src/components/CurationPage/CurationPage.tsx

```diff
@@ -142,6 +180,7 @@ export default class CurationPage extends React.PureComponent<Props, State> {
           { value: CurationFilterOptions.APPROVED, text: t('curation_page.filter.approved') },
           { value: CurationFilterOptions.REJECTED, text: t('curation_page.filter.rejected') }
         ]}
+        disabled={filterByType !== CollectionFilterOptions.STANDARD}
```

Comment:
Can we remove the _disabling_ logic? I think we support all filters for all types of collections now!

Reply:
Yes, we can. We should also remove the `reset` of the filters after changing the type, should we?